src/Card/CardImageCap.jsx

```diff
@@ -25,24 +25,27 @@ const CardImageCap = React.forwardRef(({
   const { orientation, isLoading } = useContext(CardContext);
   const [showImageCap, setShowImageCap] = useState(false);
   const [showLogoCap, setShowLogoCap] = useState(false);
+
   const wrapperClassName = `pgn__card-wrapper-image-cap ${orientation}`;
 
   if (isLoading) {
     return (
-      <div className={classNames(className, wrapperClassName)} data-testid="image-loader-wrapper">
+      <div
+        className={classNames(wrapperClassName, className)}
```

Comment:
Reverses the `wrapperClassName` and `className` such that consumers' `className` (if any) takes precedence over the component class names. Otherwise, they may be effectively ignored.

Reply:
Is this a pattern across any other components? If so, would it make sense to make an issue to standardize these?

Reply:
@brian-smith-tcril A large portion (majority) of components allows consumers to pass in a `className` prop to add custom class names or use the CSS utility classes (which rely on `!important`) to the actual class names on the component.

I'll plan to file a new GitHub issue for:
* Writing a small ADR to describe/document this standard to make it "official".
* Find other instances of this issue throughout the design system and implement a fix.

Reply:
@brian-smith-tcril I may have been wrong about this being an issue after all. I just did some test cases more explicitly about this. It appears it's not so much the order of the passed `className` needing to coming after any `.pgn__` namespaced classes.

Rather, it seems more important where the custom `className` is defined in relation to the Paragon class names. If the custom `className` is defined prior to the Paragon class names, it appears that's when consumers may run into this issue.

For example, defining a `.custom-styles-class` above all the other styles in `Card.scss` and passing `className="custom-styles-class"` to the `ActionRow` component makes no difference whether the custom `className` is ordered first or after the `.pgn__` namespaced classes in the `classNames` function.

Reply:
Glad to hear it's not an issue! Thanks for digging into that!